### DISCUSSION TITLE
-update command execution failure

### DISCUSSION THREAD

**Discussion by zan8in** [zan8in/afrog#57] [Announcements]
**Temporary solution:**

1. Need to add afrog to environment variables
2. Reopen cmd.exe (windows)
3. Execute afrog -update
4. I will fix this bug as soon as possible

**_I will fix this bug as soon as possible_**

**Comment by ViCrack** [comment]
话说这块作者可以自己在仓库里面加一个scoop、或者chocolatey的安装manifest文件，用户安装后不需要添加环境变量就可以启动，更新时也不需要afrog -update，软件都是统一管理的

**Reply by zan8in** [reply]
感谢

**Comment by cail0906** [comment]
linux 执行./main -un无法更新

**Reply by zan8in** [reply]
afrog 2.5.x 之后才新增 -un 命令，并且修复更新的BUG
